packages/astro/src/cli/add/index.ts

```diff
@@ -691,9 +691,9 @@ async function convertIntegrationsToInstallSpecifiers(
 async function resolveRangeToInstallSpecifier(name: string, range: string): Promise<string> {
 	const versions = await fetchPackageVersions(name);
 	if (versions instanceof Error) return name;
-	// Filter out any prerelease versions
-	const stableVersions = versions.filter((v) => !v.includes('-'));
-	const maxStable = maxSatisfying(stableVersions, range);
+	// Filter out any prerelease versions, but fallback if there are no stable versions
+	let stableVersions = versions.filter((v) => !v.includes('-'));
```

Comment:
Should this be left as const?

Reply:
Oops. Forgot to revert that :sweat_smile: